kb/temporal-platform-limits-sheet.md

```diff
@@ -64,3 +66,8 @@ Hard limits fail with an error; soft limits produce a warning log on the server
     - keyword: 20
     - text: 5
   - Retention Period: 30 days (configurable; range of 1–90 days)
+- **List Filters**
+  - The **ORDER BY** operator is not supported in List Filters in Temporal Cloud.
+    This means that you can not apply custom ordering of Workflows with Cloud Visibility features.
+    Lists of Workflows are still ordered by a default ordering rule.
+    Though, this rule may change in the future.
```

Comment:
```suggestion
    Be aware that this rule might change.
```
All change that hasn't yet happened occurs in the future. —Spock*

*Absolutely not Spock. Created by DailGPT.